afm-backend/repository/eventsRepository.py

```diff
@@ -109,3 +109,28 @@ def addCurrentEvent(eventId):
     cursor.close()
     db.commit()
     return cursor.lastrowid
+
+
+def delete_event(id):
+    db = Database()
+    # Delete from Current Events table
+    current_table_sql = "DELETE FROM CurrentEvent WHERE eventId=" + str(id)
```

Comment:
There should be a different end point for delete current event, this should be specific to delete events